etc/profile.d/bash_profile.sh

```diff
@@ -5,6 +5,7 @@ if [ -e ~/.bashrc -a ! -e ~/.bash_profile -a ! -e ~/.bash_login -a ! -e ~/.profi
   echo "A ~/.bash_profile that loads ~/.bashrc will be created for you."
   cat >~/.bash_profile <<-\EOF
 	# generated by Git for Windows
+	PROMPT_COMMAND='history -a'
```

Comment:
This repository actually only tracks installed Pacman packages in Git for Windows' SDK (which is a close derivative of [MSYS2](https://msys2.org/)). The actual `bash_profile.sh` lives [here](https://github.com/git-for-windows/build-extra/blob/HEAD/git-extra/bash_profile.sh).

Please open a PR there instead, and while you do so, take inspiration from https://github.blog/2022-06-30-write-better-commits-build-better-projects/ to improve the commit message (look at the existing commit messages in the `build-extra` repository to see why).